ExcelExample-FrenchTraffic/main.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+import math
+import json
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+gDEBUG = False
+gGRAPH = False
+gSAVE = True
+
+gPATHI = ""
+gPATHO = ""
+
+isDAFNI = os.environ.get("ISDAFNI")
+print("ISDAFNI Environment variable = ", isDAFNI, type(isDAFNI))
+if os.environ.get("ISDAFNI") == "True":
+    if os.name == "nt":
+        pren = os.environ.get("HOMEDRIVE")
+    else:
+        pren = "/"
+    gPATHI = os.path.join(pren, "data", "inputs")
+    gPATHO = os.path.join(pren, "data", "outputs")
+    print("Running within DAFNI: ", gPATHO)
+else:
+    print("Not running within DAFNI")
+
+print(sys.version)
+
+def trivana():
+    global gDEBUG, gGRAPH, gSAVE
+    global gPATHI, gPATHO
+    indat = os.path.join(os.path.join(gPATHI, "BAAC_2021.xlsx"))
+    print("Reading in Excel Spreadsheet with Pandas (openpyxl) : ", indat)
+    df = pd.read_excel(indat, sheet_name="Accidents corporels", header=4)
+    df = df.iloc[:, 1:]
+    if gDEBUG: 
+        print (df, df.index, df.columns)
+        if False:
+            df.plot("Année")
+    pdat = PlotData(df)
+    if gDEBUG:
+        print(pdat.dataset)
+    if gGRAPH:
+        # Quick pyplot test
+        for yr in range(1970, 1974):
+            plt.errorbar(pdat.rmonths, pdat.mvals_by_year(yr), yerr=pdat.emvals_by_year(yr), label=yr)
+        plt.legend(loc="upper left")
+        plt.show()
+        #for mnth in range(0, len(pdat.rmonths)):
+        plt.errorbar(pdat.ryears, pdat.mvals_by_mnth(7), yerr=pdat.emvals_by_mnth(7), label='Aug')
+        plt.legend(loc="upper right")
+        plt.show()
+    if gSAVE:
+        if gPATHO == "":
+            tva = "Run directory"
+        else:
+            tva = gPATHO
+        print("Saving output data to : ", tva)
+        outdaty = os.path.join(os.path.join(gPATHO, "daty.json"))
+        outdatm = os.path.join(os.path.join(gPATHO, "datm.json"))
+        jfiley = open(outdaty, "w")
+        flin1 = {}
+        flin2 = {}
+        for yr in pdat.ryears: # Mv this to the class
```

Comment:
Should the comment be removed?

There appear to be a lot of comments that could be removed